### PR TITLE
Changing to shell script (zShell Version)

### PR DESCRIPTION
**Pull request by jonstorer**
Take a look at https://github.com/thoughtbot/laptop

I'd like to make ours like this.  But also include

Checks for xcode and zShell

### CODE REVIEW

File: make.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+# RVM
+[[ -s '/Users/`whoami`/.rvm/scripts/rvm' ]] && source '/Users/`whoami`/.rvm/scripts/rvm'" >> ~/.zshrc
+  source ~/.zshrc
+
+echo "Installing Ruby 1.9.2 stable and making it the default Ruby ..."
+  rvm install 1.9.2-p290
+  rvm use 1.9.2 --default
+
+echo "Installing bundler, a tool to manage an application's dependencies through its entire life across many machines systematically and repeatably."
+  gem install bundler --no-rdoc --no-ri
```
Comment (michaeldauria) [inline make.sh:69]: I am pretty sure that rvm already includes this by default

**Comment by michaeldauria** [comment]
This is a great idea, nice work!

**Comment by quentindecock** [comment]
This is awesome

File: make.sh [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+echo "Installing ImageMagick, good for cropping and re-sizing images ..."
+  brew install imagemagick
+
+echo "Installing QT, used by Capybara Webkit for headless Javascript integration testing ..."
+  brew install qt
+
+echo "Installing the Crowdtap Profile"
+  cd ~
+  git clone git@github.com:crowdtap/dotfiles.git .dotfiles
+  cd dotfiles
```
Comment (jonstorer) [inline make.sh]: `cd .dotfiles`

**Comment by jonstorer** [comment]
close

**Pull request closed without merging**